packages/ui/src/kit/VSnackbar/VSnackbar.theme-default.scss

```diff
@@ -0,0 +1,89 @@
+.v-snackbar.theme-default {
+  display: flex;
```

Comment:
Are you sure that we should use `flex column` in that case instead `display: block`?

Reply:
yes, we need put action button (control) in bottom right place if we in `isMultiple` mode

Reply:
Why we can't enable flex in `multiple` mode only?

Reply:
in multiple will `flex` in default will `block`.
hmm, maybe stay display mode for both as equal?